### PR TITLE
Bugfix image upload

### PR DESCRIPTION
**Pull request by kedar-not-int**

### CODE REVIEW

File: packages/mdx-preview-extension/src/utils.ts [hunk 1]
```diff
@@ -371,15 +371,16 @@ export const insertImageOption = (): void => {
   let pollIterations = 0;
 
   const intervalId = setInterval(() => {
-    const destinationDiv = document
-      .getElementsByClassName("Box-sc-g0xbh4-0 hShodj")
-      .item(0);
+    const destinationDiv = document.querySelector("select[data-hasplaceholder=\"false\"]")
+      ?.parentElement
+      ?.parentElement
+      ?.parentElement;
```
Comment (agfabl) [inline packages/mdx-preview-extension/src/utils.ts:377]: looks like a risky selector as by the name it feels like a common attribute to have. @shweta-fable can you help figure out a selector that's stable and unique.

Comment (shweta-fable) [reply packages/mdx-preview-extension/src/utils.ts:377]: We can make use of the id=":rs:"
image
```
    const destinationDiv = document.getElementById(":rs:")
      ?.parentElement
      ?.parentElement
      ?.parentElement;
```

Comment (kedar-not-int) [reply packages/mdx-preview-extension/src/utils.ts]: done, used content-control el

File: packages/mdx-preview-extension/src/utils.ts [hunk 2]
```diff
@@ -371,19 +411,20 @@ export const insertImageOption = (): void => {
...
 
     pollIterations++;
 
     if (destinationDiv || pollIterations > MAX_POLL_ITERATIONS) clearInterval(intervalId);
 
-    if (destinationDiv) {
-      destinationDiv.insertAdjacentHTML("afterbegin", pictureIcon);
+    if (destinationDiv && !document.getElementById("fable-label")) {
```
Comment (siddhigate12) [inline packages/mdx-preview-extension/src/utils.ts:423]: is fable-label the image upload con? can u rename it accordingly

Comment (kedar-not-int) [reply packages/mdx-preview-extension/src/utils.ts]: done

**Pull request merged**